File: 0657-robot-return-to-origin/0657-robot-return-to-origin.java
class Solution {
    public boolean judgeCircle(String m) {
        int x=0,y=0;
     for (char i: m.toCharArray()){
        if(i=='U')x++;
        else if(i=='D')x--;
        else if (i=='R') y++;
        else if (i=='L') y--;
       
     }   
      return x==0&&y==0;
    }
}

File: 0657-robot-return-to-origin/0657-robot-return-to-origin.py
class Solution:
    def judgeCircle(self, moves: str) -> bool:
        a=0
        b=0
        for i in moves:
            if i=='U':
                a+=1
            elif i=='D':
                a-=1
            elif i=='L':
                b+=1
            elif i=='R':
                b-=1
        if a==0 and b==0:
            return True
        return False

            
            
        